Tasks/app-store-release/task.json

```diff
@@ -107,12 +107,24 @@
             "helpMarkDown": "The unique app identifier (e.g. com.myapp.etc). Required if the track is Production."
         },
         {
-            "name": "ipaPath",
+            "name": "appType",
+            "type": "pickList",
+            "label": "Application Type",
+            "helpMarkDown": "The type of application you wish to submit",
+            "required": true,
+            "options": {
+                "iOS": "iOS",
```

Comment:
Please change these to `iOS`, `tvOS` and `macOS` to match how they appear in Xcode and the Apple developer portal.